docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3'
+networks:
+  db-data-net:
+    driver: bridge
+services:
+  pythonapp:
+    image: app:1.0
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+    - "8080:8080"
+    environment:
+    - ENV=local
+    networks:
+      - db-data-net
+    depends_on:
+      - mysql
```

Comment:
The `depends_on` directive does not wait for `mysql` to be 'ready' (i.e., accepting connections). It only ensures that the `mysql` container is started before the `pythonapp` container. Consider using a script to check the database readiness or a tool like `wait-for-it`.